### PR TITLE
output-dir

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
+- Option `-o`(`--output-path`) let you specify output directory path
+- Option `ignore_groups` for optional disabling #506
```
Comment (Emilgardis) [inline CHANGELOG.md:13]: This should describe what it does, instead of simply linking to an issue

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+
 ### Changed
 
 - options can be set now with `svd2rust.toml` config
-- option `ignore_groups` for optional disabling #506
```
Comment (Emilgardis) [inline CHANGELOG.md:13]: ditto

File: src/lib.rs [hunk 3]
```diff
@@ -516,7 +516,7 @@ pub fn generate(xml: &str, config: &Config) -> Result<Generation> {
     let device = svd::parse(xml)?;
     let mut device_x = String::new();
     let items =
-        generate::device::render(&device, &config, &mut device_x).or(Err(SvdError::Render))?;
+        generate::device::render(&device, &config, &mut device_x, ".").or(Err(SvdError::Render))?;
```
Comment (Emilgardis) [inline src/lib.rs:519]: shouldn't this be configurable for the lib as well?

**Review by Emilgardis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Emilgardis) [reply CHANGELOG.md:13]: (probably don't need two changelog entries)

File: src/lib.rs (hunk 3 above)
Comment (burrbull) [reply src/lib.rs]: Added to config

**Review by Emilgardis — APPROVED** [review]
no concerns from me. bors r=me if there's nothing else

**Comment by burrbull** [comment]
bors r=Emilgardis

**Pull request merged**